src/main/java/seedu/resireg/logic/LogicManager.java

```diff
@@ -72,6 +78,16 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         return commandResult;
     }
 
+    private void addAliases(Map<String, Parser<Command>> map, List<CommandWordAlias> aliasList) {
+        for (CommandWordAlias alias : aliasList) {
+            String commandWord = alias.getCommandWord().commandWord;
+            String aliasString = alias.getAlias().alias;
```

Comment:
Similar to what Mayank said, perhaps a minor change can be made to obey CS2103's Law of Demeter?

Reply:
Yup, changed it